code/game/objects/effects/decals/misc.dm

```diff
@@ -39,4 +39,8 @@
 
 /obj/effect/decal/fake_object/light_source/invisible
 	simulated = 0
-	invisibility = 101
\ No newline at end of file
+	invisibility = 101
+
+obj/effect/decal/examine(mob/user, var/distance = -1, var/infix = "", var/suffix = "", var/show_blood = FALSE)
```

Comment:
What is the purpose of this?

Reply:
If you look at the old atom/proc/examine, it does an !istype(obj/effect/decal) to keep them from showing as "blood-stained".  When I wanted to add runes and tomes to that istype, Matt asked that I rewrite the code instead to use an argument.  So I had to rewrite the child /examine procs so that the argument positions would line up correctly.  That's why I had to add a whole new examine proc for obj/effect/decal so it could have that FALSE on the end.